docs/develop/func/cookbook.md

```diff
@@ -705,13 +763,11 @@ Note that xp+zp is a valid variable name ( without spaces between ).
    (_, int xp+zp*xp-zp) = muldivmod(xp+zp, xp-zp, prime);
```

Comment:
It should be noted that there's no need to take `xp + zp` modulo `prime`, as `muldivmod` handles it itself.
The same works for `xp - zp`: there is not even need to add `prime`, as modulo operations work mathematically in TVM (i.e. like in Python: `(-5) % 3 == 1`).